NCATS_Data_Dictionary.md

```diff
@@ -5,44 +5,59 @@
+* `scan`
   * Domains gathered from Cyber Hygiene, GSA, the End of Term Web
     Archive, or self-reported
   * SPF/DMARC/STARTTLS trustworthy email scanning
   * HTTPS web server scanning
   * SSL server scanning
-  -  Certificates and pre-certificates from [Certificate
-     Transparency](https://www.certificate-transparency.org) logs
-- `assessment` - Risk/Vulnerability Assessment (RVA) management data
+  * Certificates and pre-certificates from [Certificate
+    Transparency](https://www.certificate-transparency.org) logs
+* `assessment` - Risk/Vulnerability Assessment (RVA) management data
+* `pca` - Phishing Campaign Assessment management data
 
 This information is organized by database and collection (table).
 
 [Cyhy Database:](#cyhy-database)
-  * [cves Collection](#cves-collection)
-  * [host\_scans Collection](#host_scans-collection)
-  * [hosts Collection](#hosts-collection)
-  * [notifications Collection](#notifications-collection)
-  * [places Collection](#places-collection)
-  * [port\_scans Collection](#port_scans-collection)
-  * [reports Collection](#reports-collection)
-  * [requests Collection](#requests-collection)
-  * [snapshots Collection](#snapshots-collection)
-  * [tallies Collection](#tallies-collection)
-  * [tickets Collection](#tickets-collection)
-  * [vuln\_scans Collection](#vuln_scans-collection)
+
+* [cves Collection](#cves-collection)
+* [host\_scans Collection](#host_scans-collection)
+* [hosts Collection](#hosts-collection)
+* [notifications Collection](#notifications-collection)
+* [places Collection](#places-collection)
+* [port\_scans Collection](#port_scans-collection)
+* [reports Collection](#reports-collection)
+* [requests Collection](#requests-collection)
+* [snapshots Collection](#snapshots-collection)
+* [tallies Collection](#tallies-collection)
+* [tickets Collection](#tickets-collection)
+* [vuln\_scans Collection](#vuln_scans-collection)
 
 [scan Database:](#scan-database)
-  * [certs Collection](#certs-collection)
-  * [domains Collection](#domains-collection)
-  * [https\_scan Collection](#https_scan-collection)
-  * [precerts Collection](#precerts-collection)
-  * [sslyze\_scan Collection](#sslyze_scan-collection)
-  * [trustymail Collection](#trustymail-collection)
+
+* [certs Collection](#certs-collection)
+* [domains Collection](#domains-collection)
+* [https\_scan Collection](#https_scan-collection)
+* [precerts Collection](#precerts-collection)
+* [sslyze\_scan Collection](#sslyze_scan-collection)
+* [trustymail Collection](#trustymail-collection)
 
 [assessment Database:](#assessment-database)
-  * [assessments Collection](#assessments-collection)
-  * [findings Collection](#findings-collection)
+
+* [assessments Collection](#assessments-collection)
+* [findings Collection](#findings-collection)
+
+[PCA Database:](#pca-database)
+
+* [applications Collection](#applications-collection)
+* [assessments Collection](#assessments-collection-1)
```

Comment:
Rather than `(#assessments-collection-1)`, can you make this `(#assessments-collection-pca-db)` and update the corresponding anchor below?

If you think it's necessary for consistency, feel free also change the other one above to `(#assessments-collection-assessment-db)`.

Reply:
The anchors are auto-generated by GitHub's parser and are based on the contents of the header. `#assessments-collection-1` is what they generate for the second "assessments Collection" header because they've chosen to append numbers for headers with duplicate contents when making anchors.

Reply:
Right- I totally forgot about that!  Thanks for the follow-up.